src/DataContainer/DcaFilterInterface.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+/*
+ * This file is part of Contao.
+ *
+ * Copyright (c) 2005-2017 Leo Feyer
+ *
+ * @license LGPL-3.0+
+ */
+
+namespace Contao\CoreBundle\DataContainer;
+
+/**
+ * DCA filter interface.
```

Comment:
I would like to have some more information here so that when you read it you actually know where it's used and why (see my PR maybe :-)).